FE/src/app/components/organisation/organisation.component.scss

```diff
@@ -43,6 +43,19 @@ button {
     margin-right: 10px;
 }
 
```

Comment:
Adding

`    .flex-container {
        flex-direction: column;
        height: 442px;
    }`
    to `@media screen and (max-width:700px)`
    will allow scroll when the flex-direction is a column if we wanted to do that as well?

Reply:
good point, i missed that